src/routes/stacks/[stackId]/+page.server.ts

```
@@ -7,20 +9,56 @@
+		const servicePorts: Record<string, string[]> = {};
+		if (stack.services) {
+			for (const service of stack.services) {
+				if (service.id) {
+					try {
+						const containerData = await getContainer(service.id);
+						if (containerData && containerData.networkSettings?.Ports) {
+							const portBindings = containerData.networkSettings.Ports;
+							const parsedPorts: string[] = [];
+
+							for (const containerPort in portBindings) {
+								if (portBindings.hasOwnProperty(containerPort)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)